pkg/varlog/x/mlsa/manager.go

```diff
@@ -27,6 +27,7 @@ func (m *managedLSA) Close() {
 	appenders, ok := m.mgr.mlsas[m.tpid]
 	if ok && appenders[m.lsid] == m {
 		delete(appenders, m.lsid)
+		m.mgr.count--
```

Comment:
Are there any cases where this `managedLSA.Close` is called?

Reply:
That is up to the users. If the LogStreamAppender is failed, the user can call Close.

https://github.com/kakao/varlog/blob/b12191ac7038fbc18447099c8cfa37bd92d6bc04/pkg/varlog/log_stream_appender.go#L45-L49

Reply:
Right. `LoigStreamAppender` is interface. I missed that.